packages/bridge/README.md

```
@@ -1 +1,671 @@
+  }}
+  strictMode={true}
+/>
+```
+
+## 아키텍처
+
+### TCP 3-way Handshake
+
+Bridge는 TCP의 3-way Handshake 프로토콜을 모방하여 안정적인 통신 연결을 보장합니다.
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)